cmd/tarmak/cmd/environment.go

```diff
@@ -11,6 +12,25 @@ var environmentCmd = &cobra.Command{
 	Aliases: []string{"environment"},
 }
 
+func environmentDestroyFlags(fs *flag.FlagSet) {
+	store := &globalFlags.Environment.Destroy
+
+	fs.BoolVar(
+		&store.AutoApprove,
+		"auto-approve",
+		false,
+		"auto-approve destroy of a complete environment",
+	)
+
+	fs.StringVarP(
+		&store.Name,
+		"name",
+		"n",
+		"",
+		"name of the environment to destroy",
+	)
```

Comment:
I would prefer to have that as an argument rather than a flag

This aligns closer with `clusters set-current test` and other commands

Reply:
I agree